source/server/server.cc

```diff
@@ -320,8 +320,9 @@ void InstanceImpl::initialize(const Options& options,
   if (bootstrap_.wasm_service_size() > 0) {
     auto factory = Registry::FactoryRegistry<Configuration::WasmFactory>::getFactory("envoy.wasm");
     if (factory) {
+      auto scope = Stats::ScopeSharedPtr(stats_store_.createScope("wasm."));
```

Comment:
why do we need to create a scope called `wasm`?  If at all we do create this scope, it should record wasm implementation specific metrics.

I think that the loaded wasm modules get their own namespace with a configurable name.

Reply:
I just copied this from what another service did.  yes, perhaps we can add an additional scope which includes the VM id (which is unique for each wasm use case).